includes/class-wc-stripe-customer.php

```diff
@@ -439,7 +439,7 @@ public function get_payment_methods( $payment_method_type ) {
 		$payment_methods = get_transient( self::PAYMENT_METHODS_TRANSIENT_KEY . $payment_method_type . $this->get_id() );
 
 		if ( false === $payment_methods ) {
-			$params   = WC_Stripe_UPE_Payment_Method_Sepa::STRIPE_ID === $payment_method_type ? '?expand[]=data.sepa_debit.generated_from.charge' : '';
+			$params   = WC_Stripe_UPE_Payment_Method_Sepa::STRIPE_ID === $payment_method_type ? '?expand[]=data.sepa_debit.generated_from.charge&expand[]=data.sepa_debit.generated_from.setup_attempt' : '';
```

Comment:
Nice, that `data.sepa_debit.generated_from.setup_attempt` was the missing piece of the puzzle. This is a bit pedantic, but what would you think about moving these long expandable keys--`data.sepa_debit.generated_from.charge` and `data.sepa_debit.generated_from.setup_attempt`--to a class constant? It looks like it's getting a little crowded on this line.